### ISSUE TITLE
HTTP webhook action is in running state instead of success.

### ISSUE THREAD

**Issue by vgouth** [Azure/LogicAppsUX#6027]
### Describe the Bug with repro steps

1.Create a Stateful workflow in the Standard Logic App.
2.Open designer. And Add http request trigger and HTTP webhook action.
3.Use Method as Post , URL as Unique URL from Request Bin, Body as Callback URL token from dynamic token picker for all Subscribe and Unsubscribe parameters.
4.Save and Run the workflow.

**Expected:** Workflow Run should successful without any issues
**Actual:** Workflow run is in running state for very long period of time.

**Note:** Now the request bin URL is managed by pipedream and we have very limited access to use Unique URLs

### What type of Logic App Is this happening in?

Standard (Portal)

### Which operating system are you using?

Windows

### Are you using new designer or old designer

New Designer

### Did you refer to the TSG before filing this issue? https://aka.ms/lauxtsg

Yes

### Screenshots or Videos

![Image](https://github.com/user-attachments/assets/dacf45aa-5088-4317-b77d-f35ce832160d)
![Image](https://github.com/user-attachments/assets/1b3ab4fb-7fbd-4133-b77a-98a86028a6ef)

**Attaching HAR file for reference**
[webhook har.zip](https://github.com/user-attachments/files/17633390/webhook.har.zip)

### Browser

Edge

### Additional context

**Version :** 2.41028.1.3

**Comment by hartra344** [comment]
@vgouth did you try hitting refresh at the top? This list does not auto refresh

**Comment by vgouth** [comment]
@hartra344 , Yes travis, I tried refreshing the page and entire browser as well. Still same issue repro. Attaching screen recorder for reference.
https://github.com/user-attachments/assets/dbc23112-2edf-45c4-960b-dd3ca59c9761

**Comment by hartra344** [comment]
ok that would be a backend issue then since we're just showing what is given to us from the API. Sending to their issue list. Could you also attach a har file of the repro in an email thread?
